README.md

```diff
@@ -0,0 +1,35 @@
+## Hamiti - Neue Fische Capstone Project
+
+Hamiti is an app that helps users keep track of their food inventory and recipes.
+I am developing it as a capstone project for Neue Fische Bootcamp.
+
+## Installation
+
+To install use:
+
+```
+yarn install
+```
+
+I have opted for using yarn instead of npm, because of peer dependency issues between Material UI and Emotion.
+
+To run the development server use:
+
+```
+npm run client:dev
```

Comment:
Is it npm or yarn?☺️

Reply:
In this case, it is still npm. 
I am using yarn for managing dependencies, npm is still there to run scripts.

Reply:
So now you are only using npm right? Because it's either or :)